morpheus/stages/input/deltalake_source_stage.py

```diff
@@ -0,0 +1,146 @@
+# Copyright (c) 2021-2023, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+import os
+
+import mrc
+from pyspark.sql import SparkSession
+
+import cudf
+
+from morpheus.cli.register_stage import register_stage
+from morpheus.config import Config
+from morpheus.messages import MessageMeta
+from morpheus.pipeline.single_output_source import SingleOutputSource
+from morpheus.pipeline.stream_pair import StreamPair
+
+logger = logging.getLogger(__name__)
+
+
+@register_stage("from-deltalake")
+class DeltaLakeSourceStage(SingleOutputSource):
+    """
+    Source stage used to load messages from a DeltaLake table.
+
+    Parameters
+    ----------
+    config : morpheus.config.Config
+        Pipeline configuration instance.
+    spark_query : str
+        SQL Query that need to be executed to fetch the results from deltalake table.
+    databricks_host : str
+        URL of Databricks host to connect to.
+    databricks_token : str
+        Access token for Databricks cluster.
+    databricks_cluster_id : str
+        Databricks cluster to be used to query the data as per SQL provided.
+    databricks_port : str
+        Databricks port that Databricks Connect connects to. Defaults to 15001
+    databricks_org_id : str
+        Azure-only, see ?o=orgId in URL. Defaults to 0 for other platforms
+    """
+
+    def __init__(self,
+                 config: Config,
+                 spark_query: str,
+                 databricks_host: str = None,
+                 databricks_token: str = None,
+                 databricks_cluster_id: str = None,
+                 databricks_port: str = "15001",
+                 databricks_org_id: str = "0"):
+
+        super().__init__(config)
+        self.spark_query = spark_query
+        self._configure_databricks_connect(databricks_host,
+                                           databricks_token,
+                                           databricks_cluster_id,
+                                           databricks_port,
+                                           databricks_org_id)
+        self.spark = SparkSession.builder.getOrCreate()
+
+    @property
+    def name(self) -> str:
+        return "from-delta"
```

Comment:
```suggestion
        return "from-deltalake"
```
Match the command name